src/styles/main.css

```diff
@@ -0,0 +1,110 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card {
+  width: 198px;
+
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 32px auto 0;
+}
+
+.img {
+  display: block;
+  height: 134px;
+  width: 160px;
+}
+
+.card__description {
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.header {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 11th point